google/cloud/storage/benchmarks/storage_throughput_vs_cpu_benchmark.cc

```diff
@@ -109,6 +110,8 @@ struct Options {
       gcs_bm::ApiName::kApiGrpc,
 #endif  // GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
   };
+  std::vector<bool> enable_crc32c = {false, true};
+  std::vector<bool> enable_md5 = {false, true};
```

Comment:
The existing vector field is named `enabled_apis` with the `d`. These don't have the `d` in the name. Is that difference intentional, or do you want these to match? Note: the flag for `enable_crc32c` is named `enabled-crc32c`... could be confusing.